scipy/stats/tests/test_multivariate.py

```diff
@@ -1979,7 +1979,7 @@ def test_haar(self):
         dim = 5
         samples = 1000  # Not too many, or the test takes too long
         ks_prob = .05
-        np.random.seed(514)
+        np.random.seed(513)
```

Comment:
The preivious seed happened to fail?

Reply:
Right - does it still appear to pass approximately `1-ks_prob` of the time? 
Please check (roughly) when switching to use of a Generator (https://github.com/scipy/scipy/pull/22304/files/4938d3dcb73fc5f84399edd39db90adf90691fed#diff-a998a313f078eba79aeb6347a65117e0a0c4542a4d778a4cfd398f1737380a71)

Reply:
The test runs for 6 different comparisons, so I believe the success chance is `(1 - ks_prob)**6`. At a glance, however, it seems that everything is fine.

Reply:
Ah, I didn't notice; I meant individual tests, but that explains why it's not unlikely to fail. Sounds good.